views/index.lavender

```diff
@@ -129,12 +73,11 @@ body
                   block content  
               &lt;/body&gt;
             &lt;/html&gt;            
-  a(href='https://github.com/golavender/lavender')
+  a(href='https://github.com/golavender/lavender', target='_blank')
     img(style='position: absolute; top: 0; right: 0; border: 0;', 
      src='http://tinyurl.com/kv8nu28',
      alt='Fork me on GitHub',
-     data-canonical-src='https://s3.amazonaws.com/github/ribbons/forkme_right_white_ffffff.png')
+     data-canonical-src='https://s3.amazonaws.com/github/ribbons/forkme_right_white_ffffff.png'
+    )
 
-  footer.footer
-    .container-fluid
-      h6 Lavender is supported and created by Daft Labs. Enjoy!
\ No newline at end of file
+  include layouts/footer
```

Comment:
Instead of including the footer as a separate file on each template, you can add it after `block content` in default.lavender.